docs/tutorials/gluon/transforms.md

```diff
@@ -0,0 +1,156 @@
+dataset = dataset.transform_first(normalize_fn)
+```
+
+And now when we inspect the values for a single sample, we find that it's no longer bounded by 0 and 1.
+
+
+```python
+sample_data, sample_label = dataset[sample_idx]
+print("data range: {} to {}".format(sample_data.min().asscalar(),
+                                    sample_data.max().asscalar()))
+```
+
+### `Compose`
+
+We've now seen two examples of transforms: [`ToTensor`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=totens#mxnet.gluon.data.vision.transforms.ToTensor and `Normalize`. We applied both transforms to our `dataset` through repeated calls to `transform_first`, but Gluon has a dedicated transform to stack other transforms that's preferred. With [`Compose`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=compose#mxnet.gluon.data.vision.transforms.Compose) we can choose and order the transforms we want to apply.
+
+Caution: ordering of transforms is important. e.g. [`ToTensor`](https://mxnet.incubator.apache.org/api/python/gluon/data.html?highlight=totens#mxnet.gluon.data.vision.transforms.ToTensor) should be applied before `Normalize`, but after `Resize` and `CenterCrop`.
+
+
+```python
+dataset = mx.gluon.data.vision.CIFAR10()
+transform_fn = mx.gluon.data.vision.transforms.Compose([
+    mx.gluon.data.vision.transforms.ToTensor(),
+    mx.gluon.data.vision.transforms.Normalize(mean=[0.49139969, 0.48215842, 0.44653093],
+                                              std=[0.20220212, 0.19931542, 0.20086347]),
+])
+dataset = dataset.transform_first(transform_fn)
+```
+
+
+```python
+sample_data, sample_label = dataset[sample_idx]
+print("data range: {} to {}".format(sample_data.min().asscalar(),
+                                    sample_data.max().asscalar()))
+```
+
+As a sanity check, we get the same result as before.
+
+### `CenterCrop` and `Resize`
+
+Some networks require an input of a certain size (e.g. convolutional neural network with a final dense layer). Specifying the `size`, you can use `CenterCrop` and `Resize` to modify the spatial dimensions of an image. We show an example of downsampling to 10px by 10px here.
+
+
+```python
+dataset = mx.gluon.data.vision.CIFAR10()
+transform_fn = mx.gluon.data.vision.transforms.Resize(size=(10, 10))
+#transform_fn = mx.gluon.data.vision.transforms.CenterCrop(size=(10, 10))
+dataset = dataset.transform_first(transform_fn)
+sample_data, sample_label = dataset[sample_idx]
+plt.imshow(sample_data.asnumpy())
+```
+
+### Augmentation
+
+Augmenting samples is also common practice: that is, randomly permuting existing samples to make new samples to reduce issues of network overfitting. Using image samples as an example, you could crop to random regions, flip from left to right or even jitter the brightness of the image. Gluon has a number of random transforms that are covered in depth in the Data Augmentation tutorial.
```

Comment:
I would add a link to this tutorial: https://mxnet.incubator.apache.org/versions/master/tutorials/python/types_of_data_augmentation.html

Reply:
Going to replace this tutorial actually, so will wait till the new one is commited.